### ISSUE TITLE
Ocean is absolutely swarmed by Cachalot Whales.

### ISSUE THREAD

**Issue by OHOgameplay** [Alex-the-666/AlexsMobs#1664]
They drive the tick time to over 500 ms (basically the TPS is around 2), making the game unplayable.
This is in MC 1.19.2 with forge 43.2.0

![fjkyjrjrsjr](https://github.com/AlexModGuy/AlexsMobs/assets/124529188/90fee146-bf64-4feb-aed8-15ab970f0881)
![cgfhjfdjrzk](https://github.com/AlexModGuy/AlexsMobs/assets/124529188/6a5a3a32-2210-4d0c-9778-7ff0ca9d8877)
![gfhfhjfjg](https://github.com/AlexModGuy/AlexsMobs/assets/124529188/8fa8876a-c68e-47b8-aaa7-c72c5653aecd)

**Comment by Sansh2** [comment]
Same issue on our server with catfish. Apparently it's due to passive mob not despawning by themselvs unless you are +128 blocks away. (At least that's what one of our players explained.) If anyone finds a fix that would be great

**Comment by OHOgameplay** [comment]
> Same issue on our server with catfish. Apparently it's due to passive mob not despawning by themselvs unless you are +128 blocks away. (At least that's what one of our players explained.) If anyone finds a fix that would be great

A partial fix for me is to use the "Mob Catcher" mod. (link:https://www.curseforge.com/minecraft/mc-mods/mob-catcher)
It has a catcher "ball" and a launcher. Use the launcher just as you would use a bow. It shoots the ball, and if it hits a mob, it "captures it". You can release the captured mob by righ clicking on the ground with the ball.
For whales, or giant octopus, use the launcher (tho you can shift right click on the mob with the ball to capture, but it can crash the game for big mobs, and the laucher does not crash it).
After that, you can delete the balls, or drop them into lava.

Not the best fix, but at least it is something.

**Comment by Sansh2** [comment]
Yeah but you still have to go and seek for the mobs. We deactivated cachalot and catfish spawning and deleted and reloaded every unmodified chunks. That helped a lot. 

Also why use mob catcher ? If you have to seek the mob by yourself you might aswell use a /butcher command or something similar. I'm pretty sure it works for modded entity as well
